samples/client/petstore/kotlin/retrofit2/src/main/kotlin/org/openapitools/client/apis/UserApi.kt

```diff
@@ -10,28 +10,28 @@ import okhttp3.MultipartBody
 import org.openapitools.client.models.User
 
 interface UserApi {
-    @POST("/user")
```

Comment:
I'm not sure if this change is right or not...
Can someone else give some feedback here?

Reply:
There is still a bug in kotlin-retrofit2 generator. These releative pathes have to start WITHOUT slashes. See java-retrofit2 generator. I fixed it within my PR #5697 
https://github.com/OpenAPITools/openapi-generator/pull/5697/commits/aaef017a5a730812ac3c209cb106edda4f4349cd#diff-b60a03ff3897718b90d1c4d54850d0de in file KotlinClientCodegen.java

Reply:
Can you elaborate a bit more on why that's a bug?
I thought that "/user" was the correct behaviour.

Reply:
/user was the behaviour before my PR #5697. Retrofit2 does not allow a baseURL without trailing slash. That means, the relative path has to start without slash. The current version of kotlin-retrofit2 didnt worked in our projekt. We checked the java-retrofit2 templates. The relative pathes in the api files in java-retrofit2 didnt start with slash.

We ran into following issue: https://github.com/square/retrofit/issues/907